config/routes.rb

```diff
@@ -1,11 +1,17 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
+      namespace :items do
```

Comment:
Namespaceing was a little tricky because the "/items/find route had to go before the other resourced items routes.